pkgs/oci/inferenceql.loom/default.nix

```
@@ -0,0 +1,25 @@
+{ pkgs,
+  nixpkgs,
+  system,
+  ociImgBase,
+}: let
+  # in OCI context, whatever our host platform we want to build same arch but linux
+  systemWithLinux = builtins.replaceStrings [ "darwin" ] [ "linux" ] system;
+  crossPkgsLinux = nixpkgs.legacyPackages.${systemWithLinux};
+  python = crossPkgsLinux.python3;
+
+  scopes = (import ./../../../lib/mkScopes) pkgs;
+
+  loom = scopes.callPy3Package ./../../loom { };
+in pkgs.dockerTools.buildImage {
```

```suggestion
in pkgs.dockerTools.buildLayeredImage {
```

I suspect `buildLayeredImage` if used everywhere will have a higher chance of resulting in a smaller total size if using image inheritance (eg. `FROM`) as until the layer limit is reached there will be a 1:1 ratio of store paths to layers, and so any store paths used in both the parent and child images should be deduplicated.